src/screenbot.js

```diff
@@ -24,20 +30,56 @@ var Screenbot = (function Screenbot() {
              channel_token;
     };
 
-    var _request = function(endpoint, callback){
+    var _parseResponse = function(responseText){
+      try {
+        var response = JSON.parse(responseText);
+      }
+      catch (e) {
+        return _handleError({ code: ERROR_CODES.PARSE_ERROR.code });
+      }
+
+      return response;
+    };
+
+    var _request = function(endpoint, cb){
       var xhttp = new XMLHttpRequest();
       xhttp.onreadystatechange = function() {
         if (xhttp.readyState === 4 && xhttp.status === 200) {
-          var response = JSON.parse(xhttp.responseText);
-          callback(response.ok, response);
+          var response = _parseResponse(xhttp.responseText)
+          if(response instanceof Error || !response.ok) cb(response);
+
+          cb(null, response);
         } else if(xhttp.readyState === 4) {
-          callback(false, xhttp.responseText);
+          cb(_parseResponse(xhttp.responseText));
```

Comment:
Maybe have a comment that says something that lets the reader know that the response must be an error because the status code is not 200. At first glance, it looks like a bug because the successful `_parseResponse()` result would get sent in the callback's first (error) argument.